ClosureExterns/ClosureExternsGenerator.cs

```diff
@@ -182,15 +183,28 @@ protected StringBuilder GenerateClassDefinition(string @namespace, Type type, Qu
             typeResultBuilder.AppendLine(" */");
             typeResultBuilder.AppendLine(className + " = " + this._options.ConstructorExpression(className) + ";");
 
+            var fields = type.GetFields(BindingFlags.Static | BindingFlags.Public);
+            foreach (var fieldPair in WithIsLast(fields))
+            {
+                var field = fieldPair.Item1;
+                var mappedType = this.MapType(field.FieldType);
+                var jsTypeName = this.GetJSTypeName(type, @namespace, mappedType, graph);
+                typeResultBuilder.AppendLine($"/** @type {{{jsTypeName}}} */");
+
+                typeResultBuilder.AppendLine(
+                    $"{className}.prototype.{field.Name} = { this.GetJSValue(field.GetValue(obj: null))};");

```

Comment:
What is the result in the JS file? how does the client distinguish that it's a constant?